vtrace/notifiers.py

```diff
@@ -40,21 +40,21 @@ def handleEvent(self, event, trace):
         self.notify(event, trace)
 
     def notify(self, event, trace):
-        print "Got event: %d from pid %d" % (event, trace.getPid())
+        print("Got event: %d from pid %d" % (event, trace.getPid()))
 
 class VerboseNotifier(Notifier):
     def notify(self, event, trace):
-        print "PID %d - ThreadID (%d) got" % (trace.getPid(), trace.getMeta("ThreadId")),
+        print("PID %d - ThreadID (%d) got" % (trace.getPid(), trace.getMeta("ThreadId")))
```

Comment:
this doesn't match the same functionality.  the ',' at the end of Py2 code means "don't add a new line"

Reply:
Ah. Yea. I'll revert that for now.